### ISSUE TITLE
Multi column sorting

### ISSUE THREAD

**Issue by tkachenko1503** [kishanov/re-frame-datatable#17]
Hello! 
Is there some way to make sorting by multiple columns?

**Comment by kishanov** [comment]
Not supported ATM.

**Comment by tkachenko1503** [comment]
Thanks
Do you have some plans to implement this feature?

**Comment by kishanov** [comment]
No. I basically abandoned this project, cause at some point realized that
having component which accepts re-frame subscription and then does some
internal state management is a bad idea.
